lib/client.js

```diff
@@ -113,6 +181,64 @@ Client.prototype.dial = function (opts, cb) {
   });
 };
 
+Client.prototype.getStream = function getStream(sessionId, streamId, cb) {
+  var url = this.c.apiUrl + this.c.endpoints.getStream.replace(/<%apiKey%>/g, this.c.apiKey)
+    .replace(/<%streamId%>/g, streamId)
+    .replace(/<%sessionId%>/g, sessionId);
+  request.get({
+    url: url,
+    headers: this.generateHeaders()
+  }, function requestCallback(err, resp, body) {
+    if (err) return cb(new Error('The request failed: ' + err));
+    // handle client errors
+    switch (resp.statusCode) {
+      case 200:
+        return cb(null, body);
+      case 400:
+        return cb(new Error('Invalid session ID or token (400).'));
```

Comment:
Can we bubble up the response from the API server or hard-code the responses mimicking the API server response?